runtime/planificator.js

```diff
@@ -160,7 +226,14 @@ export default class Planificator {
     this.requestPlanning();
   }
 
+
+  onDataChanged() {
+    this._dataChangesQueue.addChange();
+  }
+
   requestPlanning(event, options) {
+    this._dataChangesQueue.onReplanDone();
```

Comment:
Is `requestPlanning` invoked once a replan has completed? That needs a comment (alternatively, `onReplanDone` needs to be renamed)

Reply:
requestPlanning is called when replanning has been requested, so renamed onReplanDone to onReplanningStarted